lib/improver/metadata/amend.py

```diff
@@ -41,27 +41,27 @@
 
 
 def update_stage_v110_metadata(cube):
-    """Translates meta-data relating to the grid_id attribute from StaGE
+    """Translates metadata relating to the grid_id attribute from StaGE
     version 1.1.0 to later StaGE versions.
     Cubes that have no "grid_id" attribute are not recognised as v1.1.0 and
     are ignored.
 
     Args:
         cube (iris.cube.Cube):
-            Cube to modify meta-data in (modified in place)
+            Cube to modify metadata in (modified in place)
 
     Returns:
-        bool:
-            True if meta-data have been changed by this function.
+        cube (iris.cube.Cube):
+            The cube with updated metadata.
```

Comment:
Suggest replacing `metadata` with `attributes` throughout this docstring (since you've made some corrections already).

Reply:
Done